lib/version.h

```diff
@@ -5,17 +5,17 @@
 #define versionH
 
 #define CPPCHECK_MAJOR_VERSION 2
-#define CPPCHECK_MINOR_VERSION 12
-#define CPPCHECK_DEVMINOR_VERSION 13
+#define CPPCHECK_MINOR_VERSION 13
+#define CPPCHECK_BUGFIX_VERSION 99
 #define CPPCHECK_FIX_VERSION 0
 
 #define STRINGIFY(x) STRING(x)
 #define STRING(VER) #VER
-#if CPPCHECK_MINOR_VERSION == CPPCHECK_DEVMINOR_VERSION
+#if CPPCHECK_MINOR_VERSION == CPPCHECK_BUGFIX_VERSION
```

Comment:
this #if checks if it's a release version.. so I suggest something like:

    #if CPPCHECK_BUGFIX_VERSION < 99

In a release I will set BUGFIX_VERSION to 0,1,2,3,...